src/dbus_api/pool.rs

```diff
@@ -485,41 +490,21 @@ pub fn create_dbus_pool<'a>(
         .emits_changed(EmitsChangedSignal::True)
         .on_get(get_pool_name);
 
-    let total_physical_size_property = f
-        .property::<&str, _>("TotalPhysicalSize", ())
-        .access(Access::Read)
-        .emits_changed(EmitsChangedSignal::False)
-        .on_get(get_pool_total_physical_size);
-
-    let total_physical_used_property = f
-        .property::<&str, _>("TotalPhysicalUsed", ())
-        .access(Access::Read)
-        .emits_changed(EmitsChangedSignal::False)
-        .on_get(get_pool_total_physical_used);
-
     let uuid_property = f
         .property::<&str, _>("Uuid", ())
         .access(Access::Read)
         .emits_changed(EmitsChangedSignal::Const)
         .on_get(get_uuid);
 
-    let state_property = f
-        .property::<u16, _>(consts::POOL_STATE_PROP, ())
-        .access(Access::Read)
-        .emits_changed(EmitsChangedSignal::True)
-        .on_get(get_pool_state);
-
-    let extend_state_property = f
-        .property::<u16, _>(consts::POOL_EXTEND_STATE_PROP, ())
-        .access(Access::Read)
-        .emits_changed(EmitsChangedSignal::True)
-        .on_get(get_pool_extend_state);
-
-    let space_state_property = f
-        .property::<u16, _>(consts::POOL_SPACE_STATE_PROP, ())
-        .access(Access::Read)
-        .emits_changed(EmitsChangedSignal::True)
-        .on_get(get_space_state);
+    let get_all_properties_method = f
+        .method("GetAllProperties", (), get_all_properties)
+        // b: true if filesystems were created
```

Comment:
Copy/paste errors in comments and return type which I'm ignoring for now.

Reply:
Resolved.